.github/workflows/publish-external-command.yml

```diff
@@ -0,0 +1,135 @@
+name: Publish External Connector Image
+on:
+  workflow_dispatch:
+    inputs:
+      connector:
+        description: "Airbyte Connector"
+        required: true
+      comment-id:
+        description: "The comment-id of the slash command. Used to update the comment with the status."
+        required: false
```

Comment:
You need to add a new input rigth?
`/publish-external harshith/source-api 0.1.0`